backends/text-to-speech/linux/linux-text-to-speech.cpp

```diff
@@ -241,6 +226,10 @@ bool SpeechDispatcherManager::say(Common::String str, Action action, Common::Str
 	return false;
 }
 
+bool SpeechDispatcherManager::say(const Common::String &str, Action action, Common::String charset) {
+	return say(Common::U32String(str), action, charset);
```

Comment:
That doesn't seem correct. The given `charset` is ignored to create the `U32String`. Doesn't it assume that this is ASCII then? This should be converted from the given `charset` to UTF-32 instead.